includes/helper/class-wp-job-manager-helper-listing-renewal.php

```diff
@@ -0,0 +1,145 @@
+<?php
+/**
+ * File containing the class WP_Job_Manager_Helper_Listing_Renewal.
+ *
+ * @package wp-job-manager
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Handles the renewal of Job Listings.
+ *
+ * @since $next-version$$
+ */
+class WP_Job_Manager_Helper_Listing_Renewal {
+
+	/**
+	 * Submit job form instance.
+	 *
+	 * @var WP_Job_Manager_Form_Submit_Job
+	 */
+	public $form = null;
```

Comment:
I don't think this should be public, especially since it's used in this class only.

Reply:
Also fixed in https://github.com/Automattic/WP-Job-Manager/pull/2422/commits/2eb330f440bcd4bf0dbb1d1d54de5306448ba041